app/instrumentation-tests/src/org/commcare/androidTests/TilesTest.kt

```diff
@@ -0,0 +1,183 @@
+package org.commcare.androidTests
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.platform.app.InstrumentationRegistry
+import org.commcare.CommCareInstrumentationTestApplication
+import org.commcare.activities.EntitySelectActivity
+import org.commcare.dalvik.R
+import org.commcare.utils.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class TilesTest: BaseTest() {
+
+    companion object {
+        const val CCZ_NAME = "tiles.ccz"
+        const val APP_NAME = "Case Tile Tests"
+        var caseDetails = arrayOf("Sex F", "Date: 01/07/17")
+        var caseDetailsExpanded = caseDetails + arrayOf("Secret")
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("tile.test", "123")
+    }
+
+    @Test
+    fun testNoTile() {
+        InstrumentationUtility.openModule(0)
+
+        caseDetails.areDisplayed()
+
+        onView(withText("Sally Ride"))
+                .perform(click())
+        withId(R.id.com_tile_holder_btn_open).doesNotExist()
+        withText("Sex F").doesNotExist()
+
+        onView(withText("Continue"))
+                .perform(click())
+        onView(withText("Placeholder"))
```

Comment:
[Step](https://github.com/dimagi/commcare-ui-tests/blob/master/features/tiles.feature#L25) missing

Reply:
Thanks for pointing it.